docs/millennium-os/manual/chapters/20_installation.md

```diff
@@ -67,64 +67,64 @@ After restarting, you will be prompted to configure MillenniumOS. This configura
 
 !!! note "Resuming the Wizard"
-    MillenniumOS now supports resuming the wizard if it was not completed. Settings are saved to an intermediate file which is loaded when the wizard is started, and you will be prompted to continue where you left off, or restart.
+    MillenniumOS now supports resuming the wizard if it is not completed. Settings are saved to an intermediate file and loaded when the wizard is started, and you will be prompted to continue where you left off or restart.
 
-If you experience any connection issues during the wizard, you can usually reload your browser tab to trigger the next dialog.
+If you experience connection issues during the wizard, you can typically reload your browser tab to trigger the next dialog.
 
 !!! tip "Datum Tools"
     You will be asked to select a Datum Tool during the wizard process, which is used to take various manual measurements.
 
-    This tool needs to be able to activate your Toolsetter, if you have one, and a reference surface (usually the machine table near where the Toolsetter is mounted) if you also have a Touch Probe.
+ This tool needs to be able to activate your Toolsetter, if you have one, and a reference surface (usually the machine table near where the Toolsetter is mounted) if you also have a Touch Probe.
 
-    It is generally easier to check this before starting the configuration wizard, as it might involve moving your spindle down in the mount to reach both of these items with a short tool.
+ It is generally easier to check this before starting the configuration wizard, as it might involve moving your spindle down in the mount to reach both of these items with a short tool.
 
-    If you realise this during the configuration wizard process, don't worry - you can resume where you left off once fixing the issue.
+ If you realize this during the configuration wizard process, don't worry - you can resume where you left off once fixing the issue.
 
 !!! note
-    If you experience any connection issues during the wizard, you can usually reload your browser tab to trigger the next dialog.
+    If you experience connection issues during the wizard, you can reload your browser tab to trigger the next dialog.
 
-You may re-run the wizard at any time to reconfigure one or more features. If you click the **"Update"** button when asked if you want to reset the configuration, then you can pick which sections of the configuration you would like to reconfigure, and all other sections of the existing configuration will be left as-is.
+You may re-run the wizard anytime to reconfigure one or more features. If you click the **"Update"** button when asked if you want to reset the configuration, you can pick which parts of the configuration you would like to reconfigure. All other sections of the existing configuration will be left as-is.
```

Comment:
Replaced parts in one sentence but not the other. Would prefer we use one or the other consistently, no strong opinion which one.